presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/RewriteAggregationIfToFilter.java

```diff
@@ -199,4 +225,23 @@ private boolean shouldRewriteAggregation(Aggregation aggregation, ProjectNode so
         // Only rewrite the aggregation if the else branch is not present or the else result is NULL.
         return expression.getForm() == IF && Expressions.isNull(expression.getArguments().get(2));
     }
+
+    private boolean canUnwrapIf(SpecialFormExpression ifExpression)
+    {
+        boolean[] result = {true};
```

Comment:
Usually we don't use array for reference; let's switch to AtomicBoolean

Reply:
Done.